tests/integration/cmd_namespace_test.go

```diff
@@ -22,13 +22,13 @@ var _ = Describe("odo create/delete/list/set namespace/project tests", func() {
 	AfterEach(func() {
 		helper.CommonAfterEach(commonVar)
 	})
-
 	for _, commandName := range []string{"namespace", "project"} {
-
+		// this is a workaround to ensure that the for loop works with `It` blocks
+		commandName := commandName
```

Comment:
You are creating shadowed variables this way. It would be better to use different names for the two variables

Reply:
> This will generate several Its - one for each category. Note that you must assign a copy of the loop variable to a local variable (that's what category := category is doing) - otherwise the It closure will capture the mutating loop variable and all the specs will run against the last element in the loop. It is idiomatic to give the local copy the same name as the loop variable.(https://onsi.github.io/ginkgo/#dynamically-generating-specs)

It is by design if I am understanding it correctly.

Reply:
I was also surprised to see that this was not reported by `golangci-lint` (and `govet`) as a potential issue.
I guess this is particularly fine for `govet` as we are shadowing the variable with itself.
It would be detected by `golangci-lint` if we assigned a different value, I think.